spec/support/env.cr

```diff
@@ -1,9 +1,9 @@
 def with_env(values : Hash)
   old_values = {} of String => String?
   begin
+    old_values = ENV.to_h
```

Comment:
Stash all old values, not just ones related to the `values` argument.
This should let `with_env` work with methods that remove entries from the ENV like delete, clear, replace, etc.

Reply:
This line should be merged with the previous assignment. It isn't supposed to be in the `begin` block.